build.gradle

```diff
@@ -356,171 +364,3 @@ distZip {
         delete fileTree(dir: 'build/distributions', include: '*.zip')
     }
 }
-
-def dockerBuildVersion = project.hasProperty('release.releaseVersion') ? project.property('release.releaseVersion') : "${rootProject.version}"
-def dockerOrgName = project.hasProperty('dockerOrgName') ? project.getProperty("dockerOrgName") : "consensys"
-def dockerArtifactName = project.hasProperty("dockerArtifactName") ? project.getProperty("dockerArtifactName") : "linea-shomei"
-def dockerImageName = "${dockerOrgName}/${dockerArtifactName}"
-def dockerVariants = project.hasProperty("dockerVariants") ? project.getProperty("dockerVariants").split(",") : [
-        "openjdk-21",
-        "openjdk-21-debug",
-        "openjdk-latest"
-]
-def archVariants = ["amd64", "arm64"]
-
-// rename the top level dir from shomei-<version> to shomei and this makes it really
-// simple for use in docker
-task dockerDistUntar {
-    dependsOn distTar
```

Comment:
It looks like the impact of doing this means we only have CI/github-actions built docker versions.  we won't be able to generate a docker image for local testing, or debugging or patching if we remove this

Reply:
I can add this in as a seperate task but the rationale here is to use the right tool for the job and standardizing things, in this its 10x work to create a docker image + manifest, which is overkill. Also the benefit of having any/every commit push create artifacts means you don't need to hand roll stuff because the ci does this

Reply:
long weekend - I see this is merged already, but just to follow up.  I debug using docker containers often.  when I need to run a test I usually have compose files that make the task of doing debug sync or rpc testing or whatver just a matter of just building a docker image and starting the compose.    That flow is broken if I can't create an image outside of merging a commit to main.  

it isn't the end of the world for shomei, since it is not too often I need to do this kind of work, but I will make a much bigger fuss if we want to remove this ability from core besu.  it is just too useful and necessary there.  especially regarding tooling like rocketpool, eth-docker, dappnode, etc

Reply:
All good mate, understand the part about docker. Atm I don't think besu fits into what we can streamline and standardize given they're another org? For our group though its getting too hard to stay consistent with n plugins and m test methods and such. 

Specifically here though I think we could put the compose files directly in code and build via compose directly? `docker-compose build ...` I don't think we need to scrap this functionality, just get into repeatable patterns across the repos - Im willing to bet that there is other folk doing this via compose as well? My preference is to use the right tool for the job (as opposed to gradle and bash for everything) and keep it consistent across repos.

Reply:
Something like this https://github.com/Consensys/web3signer/tree/master/docker even if its plain ole docker